stingray/lightcurve.py

```diff
@@ -162,9 +215,26 @@ def __add__(self, other):
                              "of same dimension and equal.")
 
         new_counts = np.add(self.counts, other.counts)
+
+        if self.statistic != other.statistic:
+            simon("Lightcurves have different statistics!"
+                  "We are setting the errors to zero to avoid complications.")
+            new_counts_err = np.zeros_like(new_counts)
+        elif self.statistic in ['poisson', 'gaussian']:
```

Comment:
This only works in the Gaussian regime, or when count rates are high for Poissonian light curves. This will fail if we decide to use anything else than `sqrt` to calculate the Poissonian errors.